Makefile

```diff
@@ -428,4 +435,25 @@ vendor:
 
 .PHONY: install-go-tools
 install-go-tools:
-	go install ${GOTESTSUM}
\ No newline at end of file
+	go install ${GOTESTSUM}
+
+AZURE_PREFIX ?= zzz
+RP_FULL_DEV_IMAGE ?= generic-repo/rp-full-dev:v0.0.1
+ARO_RP_BRANCH ?= razo7/ARO-9327
+SKIP_DEPLOYMENTS ?= true
```

Comment:
current convention is to put Makefile vars at the top of the file - can we move these there?

Reply:
NP